syntax/encode_test.go

```diff
@@ -3,10 +3,50 @@ package syntax
 import (
 	"bytes"
 	"encoding/hex"
+	"fmt"
 	"strings"
 	"testing"
 )
 
+type CrypticString string
+
+var (
+	crypticStringMarshalCalls   = 0
+	crypticStringUnmarshalCalls = 0
+)
```

Comment:
These seem to be unused.

Reply:
Added references to the tests, which I had forgotten.